### PR TITLE
koord-manager: add README for noderesource framework

### PR DESCRIPTION
**Pull request by saintube**
### Ⅰ. Describe what this PR does

koord-manager: Add a README for the node resource framework.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/slo-controller/noderesource/framework/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+**NeedSync**:
+
+The plugin checks the extended resources `kubernetes.io/batch-cpu`, `kubernetes.io/batch-memory` of the prepared node
+and the old node. If the node's Batch resources have not been updated for too long or the calculated results changes
+too much, it will update the prepared Node object to the kube-apiserver.
+
+## What's More
+
+The node resource framework is in *Alpha*. The defined stages may be not enough for some new scenarios. Please feel
```
Comment (jasonliu747) [inline pkg/slo-controller/noderesource/framework/README.md:53]: may be not -> may not be

Comment (saintube) [reply pkg/slo-controller/noderesource/framework/README.md]: fixed

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/slo-controller/noderesource/framework/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+The node resource controller is responsible for calculating the over-commit resources and updating the result on the
+node. The node resource framework defines some extension points for resource calculation and updating strategy. Each
+node resource plugin can implement one or more stages to reconcile the node resources.
+
+The current extension stages provided by the node resource framework are as following:
+
+- **Setup**: It setups the plugin with options like controller client, scheme and event recorder.
+- **Calculate**: It calculates the node resources according to the Node and NodeMetric. In case of the NodeMetric is
+abnormal, a plugin can implement the degraded calculation inside this stage.
+- **Reset**: It resets the node resources when the colocation is configured as disabled.
```
Comment (hormes) [inline pkg/slo-controller/noderesource/framework/README.md:14]: Calculate and reset seem to be the same extension point, how to understand the difference?

Comment (saintube) [reply pkg/slo-controller/noderesource/framework/README.md]: Fixed. The Reset stage is removed, and the reset is considered a method of the Calculate stage.

**Pull request merged**